ios/engine/KMEI/KeymanEngine/Classes/Extension/JSONDecoder.DateDecodingStrategy+ISO8601Fallback.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  JSONDecoder.DateDecodingStrategy+ISO8601Fallback.swift
+//  KeymanEngine
+//
+//  Created by Gabriel Wong on 2017-11-16.
+//  Copyright © 2017 SIL International. All rights reserved.
+//
+
+import Foundation
+
+extension JSONDecoder.DateDecodingStrategy {
+  static var ios8601WithFallback: JSONDecoder.DateDecodingStrategy {
+    if #available(iOS 10.0, *) {
+      return .iso8601
+    }
+    let formatter = DateFormatter()
+    formatter.locale = Locale(identifier: "en_US_POSIX")
+    formatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ssZZZZZ"
```

Comment:
Do you need to define timeZone? UTC?

Reply:
I believe the `ZZZZZ` specifies time zone.

For reference, I took the format from [here](https://stackoverflow.com/questions/16254575/how-do-i-get-an-iso-8601-date-on-ios). Note the caveat that this format doesn't deal with milliseconds, but the API doesn't provide them either from what I observed.